### ISSUE TITLE
README.md: Should our minimum CMake/Ninja versions be those installed by VS?

### ISSUE THREAD

**Issue by StephanTLavavej** [microsoft/STL#762]
Currently, the README says:

> Install CMake 3.16.5 or later, Ninja 1.10.0 or later, and Visual Studio 2019 16.6 Preview 2 or later.

I observe that VS 2019 16.6 Preview 4 installs CMake 3.17 (apparently a slightly modified version) and Ninja 1.8.2, and that these can be used to successfully build the STL:

```
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.6.0-pre.4.0
** Copyright (c) 2020 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'

C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview>where cmake
C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe

C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview>cmake --version
cmake version 3.17.20032601-MSVC_2

CMake suite maintained and supported by Kitware (kitware.com/cmake).

C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview>where ninja
C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja\ninja.exe

C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview>ninja --version
1.8.2
```

For the record, I really **really** like using the latest dependencies. However, given that the VS installer installs CMake and Ninja, should these be our minimum required versions? It seems likely to me that many people will use them anyways even if we try to require newer versions.

This would also allow us to simplify our instructions; we wouldn't need to mention installing CMake and Ninja separately.

**Comment by cbezault** [comment]
Whatever we test with is what we should require to build.
That said, we can easily decide to pin the version of CMake and Ninja that we put on the CI machines to what is offered in the most recent VS release. Luckily they're somewhat aggressive in upgrading.

**Comment by StephanTLavavej** [comment]
This was fixed by #1153 - since then, we've recommended that users get CMake/Ninja from VS, and otherwise the VS versions are our minimum required versions.
